running-dapr/releases/release-process.md

```diff
@@ -1,9 +1,8 @@
 # Release cycle and cadence 
```

Comment:
If you are editing this file, I would put an intro paragraph here to the release cycle purpose. This is currently missing.